README.md

```diff
@@ -86,23 +87,24 @@ Additionally, you can pass some extra options to the `new` command. Type `codelo
 
 Once all changes were merged and the release is ready to be packed, all that must be done is to run the following command:
 
-``` bash
+```bash
 $ codelog release [VERSION] <RELEASE_DATE>
 ```
-Where the `VERSION` is mandatory and represents the number of the next version of the system. An additional parameter `RELEASE_DATE` can be used, and if so, the release date of the version will be that value. You can configure the format of this parameter in the configuration file, under `changelogs/codelog.yml`.
 
-No conflicts to resolve. All changes documented.
+Where the `VERSION` is mandatory and represents the number of the next version of the system. An additional parameter `RELEASE_DATE` can be used, and if so, the release date of the version will be that value. You can configure the format of this parameter in the configuration file, under `changelogs/codelog.yml`.
 
-It will execute 3 steps:
+This command will execute 3 steps:
 
 - Generates a new release file at `changelogs/releases/` by merging all change files at `changelogs/unreleased/`
-- Deletes the change files at `changelogs/unreleased/` because they now compose the new release. If it was not deleted, the change would appear repeated in the next release.
+- Deletes the change files at `changelogs/unreleased/` because they now compose the new release.(if these files are not deleted, the same changes would appear in the next release.
```

Comment:
Looks like the parentheses are being opened but are not being closed on line 99. I would also change the `if these files are not deleted` to `Otherwise`

Reply:
```suggestion
- Deletes the change files at `changelogs/unreleased/` because they now compose the new release.(Otherwise, the same changes would appear in the next release).
```